lib/rubocop/cop/rspec_rails/base.rb

```diff
@@ -19,7 +18,7 @@ def self.inherited(subclass) # rubocop:disable Lint/MissingSuper
         # that have no other means of accessing the configuration.
         def on_new_investigation
           super
-          RuboCop::RSpec::Language.config = config['RSpec']['Language']
+          RuboCop::RSpec::Language.config = config['RSpecRails']['Language']
```

Comment:
Shouldn’t this be kept to pick up rhe conf as defined by the default conf or rubocop-rapec (and only optionally augmented by other gems and project config)?

Reply:
Indeed! Thank you!